### PR TITLE
Releasing component SDK

### PR DESCRIPTION
**Pull request by hongye-sun**
The SDK is shipped with:
1. A python package called kfp-component.tar.gz in the release folder
2. A container called ml-pipeline-gcp with the sdk installed

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/842)

### CODE REVIEW

File: components/release.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ images=(
   "resnet-train"
   "ml-pipeline-local-confusion-matrix"
   "ml-pipeline-local-roc"
+  "ml-pipeline-gcp"
```
Comment (vicaire) [inline components/release.sh]: Did you get through the review process for new images?

Comment (hongye-sun) [reply components/release.sh]: Thanks for the reminder. I just submitted the request.

Comment (vicaire) [reply components/release.sh]: Thanks. Let me know when it is accepted.

**Pull request merged**